odh/base/monitoring/github-receiver-service.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: github-receiver-service
+spec:
+  ports:
+    - port: 9393
+      protocol: TCP
```

Comment:
TCP is already default, I don't think you need to specify it just the `port` and `targetport` are sufficient